test/lib/mayaUsd/render/vp2RenderDelegate/testVP2RenderDelegateSelection.py

```diff
@@ -130,12 +130,12 @@ def testSelection(self):
         cmds.modelEditor('modelPanel4', e=True, wireframeOnShaded=False, displayLights='default')
         self._selectionTest('', usdCube, usdCylinder, proxyDagPath, 'smoothShaded')
 
-        # Test smooth shaded
+        # Test wireframe on shaded
```

Comment:
I noticed these comment typos when I was working on these changes getting the test to work correctly. The test behavior itself does not change.